From f2ac37b0b611a2c38ca19bf0b104cfe9ce15312b Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Thu, 26 Jul 2018 16:55:12 +0200
Subject: [PATCH] Use packaged versions of header-only libraries.

---
 CMakeLists.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 948942a..3f1aa34 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,16 +18,6 @@ if(BUILD_DOC)
   include(Doxygen)
 endif()
 
-file(DOWNLOAD
-       "https://github.com/nlohmann/json/releases/download/v3.1.2/json.hpp"
-       ${PROJECT_SOURCE_DIR}/include/json.hpp
-     EXPECTED_HASH
-       SHA256=fbdfec4b4cf63b3b565d09f87e6c3c183bdd45c5be1864d3fcb338f6f02c1733)
-
-file(DOWNLOAD
-       "https://github.com/mpark/variant/releases/download/v1.3.0/variant.hpp"
-       ${PROJECT_SOURCE_DIR}/include/variant.hpp
-     EXPECTED_MD5 "be0ce322cdd408e1b347b9f1d59ea67a")
 
 set(SRC
     src/events.cpp
-- 
2.17.1

