From c115757e8054f8468cd1272289f9ad33ef33ce39 Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Mon, 18 Jun 2018 20:28:40 +0200
Subject: [PATCH] Added SONAME version to shared library.

Signed-off-by: Vitaly Zaitsev <vitaly@easycoding.org>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cbb2274..c9bb152 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -194,3 +194,5 @@ if(BUILD_TESTS)
   add_test(Responses responses)
   add_test(Requests requests)
 endif()
+
+set_property(TARGET matrix_structs PROPERTY SOVERSION ${PROJECT_VERSION})
-- 
2.17.1

