From 567d8de2cdfe12ef876b0c41c4f7a744e0ef649f Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Tue, 10 Oct 2017 17:18:17 +0200
Subject: [PATCH] Removed extra CXXFLAGS.

---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3473043..539504e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -94,12 +94,6 @@ message(STATUS "Version: ${PROJECT_VERSION}")
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} \
-        -Wall \
-        -Wextra \
-        -march=native \
-        -Werror \
-        -pipe \
-        -pedantic \
         -Wunreachable-code")
 
     if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
-- 
2.13.6

