From 1483ea4d2d86af4804441406a838fdf0af082c3a Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Tue, 12 Dec 2017 15:24:19 +0100
Subject: [PATCH 1/2] Drop Git submodules from cmake manifest.

---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9a0a136..1db1066 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,11 +43,6 @@ else()
     find_package_handle_standard_args(LMDB DEFAULT_MSG LMDB_INCLUDE_DIR LMDB_LIBRARY)
 endif()
 
-if (NOT EXISTS "${CMAKE_SOURCE_DIR}/libs/lmdbxx/.git" )
-    message(WARNING "The git submodules are not available.")
-    message(STATUS "Running git submodule update --init --recursive ...")
-    execute_process(COMMAND git submodule update --init --recursive)
-endif()
 
 #
 # Discover Qt dependencies.
-- 
2.14.3

