From 5d4e7aad8451beaef6941850e50c081732fdd5df Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Sat, 16 Dec 2017 18:20:43 +0100
Subject: [PATCH 2/2] Remove rpath.

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e397d67..e46f033 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -365,3 +365,4 @@ if(UNIX AND NOT APPLE)
             COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
     endif()
 endif()
+set_target_properties(nheko PROPERTIES SKIP_BUILD_RPATH TRUE)
-- 
2.14.3

