From 6e606f1637e93619feae69bf1eae8dace7d4900d Mon Sep 17 00:00:00 2001
From: Vitaly Zaitsev <vitaly@easycoding.org>
Date: Tue, 12 Dec 2017 15:25:45 +0100
Subject: [PATCH 2/3] Completely remove additional CXXFLAGS.

---
 CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ecf9645..e481b11 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -111,13 +111,6 @@ set(PROJECT_VERSION ${CPACK_PACKAGE_VERSION})
 message(STATUS "Version: ${PROJECT_VERSION}")
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} \
-        -Wall \
-        -Wextra \
-        -Werror \
-        -pipe \
-        -pedantic \
-        -Wunreachable-code")
 
     if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
         execute_process(COMMAND ${CMAKE_CXX_COMPILER} -dumpversion OUTPUT_VARIABLE GCC_VERSION)
-- 
2.14.3

